tests/ops/qubit/test_parametric_ops.py

```diff
@@ -2105,13 +2117,17 @@ def test_pcphase_eigvals(self):
         # test arbitrary phase shift
         phi = 0.5432
         op = qml.PCPhase(phi, dim=2, wires=[0, 1])
-        expected = np.array([np.exp(1j * phi), np.exp(1j * phi), 1.0, 1.0])
+        expected = np.array(
+            [np.exp(1j * phi), np.exp(1j * phi), np.exp(-1j * phi), np.exp(-1j * phi)]
+        )
         assert np.allclose(op.eigvals(), expected)
 
         # test arbitrary broadcasted phase shift
         phi = np.array([0.5, 0.4, 0.3])
         op = qml.PCPhase(phi, dim=2, wires=[0, 1])
-        expected = np.array([[np.exp(1j * p), np.exp(1j * p), 1.0, 1.0] for p in phi])
+        expected = np.array(
```

Comment:
How this is computed? If computed by hand or it is a trivial solution, might add a comment to clarify.